content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>You determine these constants for Moore's law by specifying the rate for added <strong>transistors</strong>, 2 (missing a t in transistors)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
spell-check, thanks. I misspelled _semiconduct~~e~~or_ right above too.